threadpoolctl.py

```diff
@@ -523,6 +525,24 @@ def _check_prefix(self, library_basename, filename_prefixes):
                 return prefix
         return None
 
+    def _warn_if_incompatible_openmp(self):
+        """Raise a warning if llvm-OpenMP and intel-OpenMP are both loaded"""
+        if sys.platform != 'linux':
+            # Only raise the warning on linux
+            return
+
+        prefixes = [module.prefix for module in self.modules]
+        msg = textwrap.dedent(
+            """
+            Found intel OpenMP ('libiomp') and llvm OpenMP ('libomp') loaded at
+            the same time. Both libraries are known to be incompatible.
```

Comment:
and this can cause random crashes or deadlocks on Linux when loaded in the same Python program.